pkg/addons/data/Namespace/dedicated-admin.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: dedicated-admin
```

Comment:
Why do we need this and the `dedicated-readers` namespaces?

Reply:
Explanation above.